tasks/gcp/SNPGenotypingTasks.wdl

```diff
@@ -74,11 +74,12 @@ task UnifiedGenotyper {
 task VcfToZarr {
   input {
     File input_vcf
+    File input_vcf_index
```

Comment:
Does sample_vcf_to_zarr.py *require* a VCF index? If not, it might be better to make this an optional input and only touch the file if it exists.

Reply:
Yes, it does now require a VCF index as it uses tabix to access the data for each contig.